website/docs/02-About/3-branding.mdx

```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
 
 **Moja global** is always written in all lower case, except when at the start of a sentence in which case it appears as **Moja global**. Moja global is written in lower case because **open source** is also the lower case (even though both are registered trademarks). While there is no published reason why moja global uses lower case, some have explained the use of lower case as a contrast to proper nouns and as an expression of the supportive role moja global tries to fulfill.
 
-### Logo
+## Logomarks
 
-| Name | Image |
-| --- | --- |
-| Logo (Transparent Background) | <img src="/img/moja-global-logo-transparent-background.png" width="100"/> |
-| Logo (White Background) | <img src="/img/moja-global-logo-white-background.png" width="100"/> |
+### Primary Logomark
```

Comment:
`Logomark` should be in smallcase. Headings should always have the first character in the uppercase alone.